### PR TITLE
Fix typo in web3-utils.rst

### PR DESCRIPTION
**Pull request by eip1599**
## Description

Fixes typos ("ethere unit") in web3-utils documentation.

## Type of change

Fix typos. "ethere unit" => "ether unit"

### CODE REVIEW

File: docs/web3-utils.rst [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ toWei
 
 Converts any `ether value <http://ethdocs.org/en/latest/ether.html>`_ value into `wei <http://ethereum.stackexchange.com/questions/253/the-ether-denominations-are-called-finney-szabo-and-wei-what-who-are-these-na>`_.
 
-.. note:: "wei" are the smallest ethere unit, and you should always make calculations in wei and convert only for display reasons.
+.. note:: "wei" are the smallest ethereum unit, and you should always make calculations in wei and convert only for display reasons.
```
Comment (Spacesai1or) [inline docs/web3-utils.rst]:
```suggestion
.. note:: "wei" are the smallest ether unit, and you should always make calculations in wei and convert only for display reasons.
```

File: docs/web3-utils.rst [hunk 2]
```diff
@@ -1117,7 +1117,7 @@ fromWei
 
 Converts any `wei <http://ethereum.stackexchange.com/questions/253/the-ether-denominations-are-called-finney-szabo-and-wei-what-who-are-these-na>`_ value into a `ether value <http://ethdocs.org/en/latest/ether.html>`_.
 
-.. note:: "wei" are the smallest ethere unit, and you should always make calculations in wei and convert only for display reasons.
+.. note:: "wei" are the smallest ethereum unit, and you should always make calculations in wei and convert only for display reasons.
```
Comment (Spacesai1or) [inline docs/web3-utils.rst]:
```suggestion
.. note:: "wei" are the smallest ether unit, and you should always make calculations in wei and convert only for display reasons.
```

**Review by Spacesai1or — CHANGES_REQUESTED** [review]
According to the [Stack Overflow post](https://ethereum.stackexchange.com/questions/253/the-ether-denominations-are-called-finney-szabo-and-wei-what-who-are-these-na) (mentioned where these changes are being made), I believe the typo, `ethere unit`, should've been corrected to `ether unit` instead of `ethereum unit`

File: docs/web3-utils.rst (hunk 1 above)
Comment (eip1599) [reply docs/web3-utils.rst]: Nice catch! Would be even better to use "unit of ether"?

**Review by frankiebee — APPROVED** [review]
thanks!

**Review by Spacesai1or — APPROVED** [review]

**Pull request merged**